src/strategies/FillStateParams.ts

```diff
@@ -0,0 +1,32 @@
+import { FillStateParamsJSON, FillStateParamsArgs, BigIntish, FillStateParamsStruct } from '@brinkninja/types'
+
+class FillStateParams {
+  id: BigIntish
+  sign: boolean
+  startX96: BigIntish
+
+  public constructor (args: FillStateParamsArgs) {
+    this.id = args.id
```

Comment:
we should probably throw an error if id is not set .. `if (!args.id) {...` should work to check for undefined, null, or 0. error like "id required for FillStateParams" would work.

I didn't add this to the Token class but probably should...